container_service_extension/client/utils.py

```diff
@@ -7,9 +7,27 @@
 from vcd_cli.profiles import Profiles
 
 from container_service_extension.client import system as syst
+from container_service_extension.client.constants import CSE_SERVER_RUNNING
 import container_service_extension.def_.utils as def_utils
 from container_service_extension.shared_constants import CSE_SERVER_API_VERSION
 
+_RESTRICT_CLI_TO_TKG_OPERATIONS = False
```

Comment:
If true, can we show the only _cluster apply, info, delete and list commands_? It should be straight-forward, please check. If not, you can skip it.

(sorry for the late comment, it just occurred to me).